session6/src/com/itschool/homework/HomeRunchallenge6.java

```diff
@@ -0,0 +1,18 @@
+package com.itschool.homework;
+
+public class HomeRunchallenge6 {
```

Comment:
typo in class name. Rename it to `HomeRunChallenge6` (uppercase C in Challenge): in the left pane where you have the project structure (Project view) right click on the class's name -> Refactor -> Rename. This will change both the class name here and the java file name (this is important because the java file name needs to match exactly the class name in the code).